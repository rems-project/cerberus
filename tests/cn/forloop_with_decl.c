int for_with_decl()
{
  int acc = 0;
  for(int i = 0; i < 10; i++)
  /*@ inv 0i32 <= i; i <= 10i32;
          acc <= 10i32; @*/
  {
    acc = i;
  };
  return acc;
}

int main(void)
/*@ trusted; @*/
{
  int r = for_with_decl();
}