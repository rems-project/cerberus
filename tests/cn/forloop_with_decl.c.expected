return code: 0
[1/1]: for_with_decl -- pass
